internal/workflow.go

```diff
@@ -415,6 +415,7 @@ func ExecuteLocalActivity(ctx Context, activity interface{}, args ...interface{}
 	params := &executeLocalActivityParams{
 		localActivityOptions: *options,
 		ActivityFn:           activity,
+		ActivityType:         lastPartOfName(getFunctionName(activity)),
```

Comment:
Can we register local activity like we do for regular one?
I'm little worried that using last part of the name is might not be enough as it will still break replay if someone renames a function.

Reply:
We do the same for regular activity. The intention is that in case user move code around (from one package to another package), we don't want to break their workflow.

Reply:
Register would not help, unless your enforce that every local activity should be registered with a friendly name. Even that would not prevent problem because user could still update the registered name.